expression/builtin_time.go

```diff
@@ -2721,6 +2721,10 @@ func (du *baseDateArithmitical) getDateFromString(ctx sessionctx.Context, args [
 
 	sc := ctx.GetSessionVars().StmtCtx
 	date, err := types.ParseTime(sc, dateStr, dateTp, types.MaxFsp)
+
```

Comment:
Should the err be considered  `err != nil` first?

Reply:
done